end-to-end-test/local/runtime-config/serve_dist.sh

```diff
@@ -4,10 +4,9 @@ set -e
 set -u # unset variables throw error
 set -o pipefail # pipes fail when partial command fails
 
-(echo $CBIOPORTAL_URL | grep -q https) \
-&& ( \
-    openssl \
-        req -newkey rsa:2048 -new -nodes -x509 -days 1 -keyout key.pem -out cert.pem \
-        -subj "/C=US/ST=Denial/L=Springfield/O=Dis/CN=localhost" && \
-    ./node_modules/http-server/bin/http-server -S -C cert.pem --cors dist/ -p 3000 \
-) || ./node_modules/http-server/bin/http-server --cors dist/ -p 3000
+
```

Comment:
@zainasir this is all i had to change.  this makes sure frontend server always uses https.  which corresponds to change made in backend as part of rfc80.